openpype/plugins/publish/integrate.py

```diff
@@ -529,6 +529,13 @@ def prepare_representation(self, repre,
         template_data["representation"] = repre["name"]
         template_data["ext"] = repre["ext"]
 
+        # add template data for colorspaceData
+        if repre.get("colorspaceData"):
+            colorspace = repre["colorspaceData"]["colorspace"]
+            # replace spaces with underscores
+            colorspace = colorspace.replace(" ", "_")
```

Comment:
I'm not sure if the replacement of spaces should happen.

Reply:
yes it should. I am retreaving the file colorspace with underscores here https://github.com/pypeclub/OpenPype/pull/4270/files#diff-c4b595d393443040b1d7c9551351bfffe69f9c00ad87bd9b6243da47a643f337R118

Reply:
I would add it to the code comment then, otherwise it will get lost in time.